### PR TITLE
Add MACE

### PR DESCRIPTION
**Pull request by janosh**
As released in [NeuRIPS paper](https://proceedings.neurips.cc/paper_files/paper/2022/hash/4a36c3c51af11ed9f34615b81edb5bbc-Abstract-Conference.html) and further developed in [2022 preprint](https://arxiv.org/abs/2205.06643).

This submission uses the [`2023-07-14-mace-universal-2-big-128-6.model`](https://figshare.com/ndownloader/files/41565618) checkpoint pre-trained by the MACE authors on the [original M3GNet training set](https://figshare.com/articles/dataset/MPF_2021_2_8/19470599).

MACE relaxed each test set structure using [`ase.optimizers.FIRE`](https://wiki.fysik.dtu.dk/ase/ase/optimize.html#fire) until the maximum force in the training set drops below 0.05 eV/Å or 500 optimization steps are reached, whichever occurs first.

### CODE REVIEW

File: matbench_discovery/plots.py [hunk 1]
```diff
@@ -875,38 +876,80 @@ def df_to_svelte_table(
...
+        file_path (str | Path): Path to the PDF file.
+    """
     try:
-        # pdfkit used to export pandas Styler to PDF, requires:
-        # pip install pdfkit && brew install homebrew/cask/wkhtmltopdf
-        import pdfkit
-    except ImportError as exc:
-        raise ImportError(
-            "pdfkit not installed\nrun pip install pdfkit && brew install "
-            "homebrew/cask/wkhtmltopdf\n(brew is macOS only, use apt on linux)"
```
Comment (janosh) [inline matbench_discovery/plots.py:893]: Reason: `wkhtmltopdf` requires Rosetta, i.e. is not natively M1 Mac compatible.
